apps/react-storybook/stories/utils/wrappers.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useRef, useEffect } from "react";
+import * as inferno from "inferno";
+
+export function wrapInfernoWithReact<TProps>(component) {
```

Comment:
Where is this function used?

Reply:
In this PR it is not used (we don't have inferno components yet)
in CardView draft branch it is used for Card story: https://github.com/DevExpress/DevExtreme/blob/13b881b23183ee291be9a44992690de9d15f968c/apps/react-storybook/stories/card_view/Card.stories.tsx#L20